### PR TITLE
Initial commit of System.Globalization.Extensions

### PR DESCRIPTION
**Pull request by Priya91**

### CODE REVIEW

File: src/System.Globalization.Extensions/tests/IdnMapping/Data/Unicode_6_0/IdnaTest.txt [hunk 1]
```diff
@@ -0,0 +1,850 @@
...
+N;	βόλος;	;	xn--nxasmm1c
+B;	ΒΌΛΟΣ;	βόλοσ;	xn--nxasmq6b
+B;	βόλοσ;	;	xn--nxasmq6b
+B;	Βόλοσ;	βόλοσ;	xn--nxasmq6b
+B;	xn--nxasmq6b;	βόλοσ;	xn--nxasmq6b
+B;	XN--NXASMQ6B;	βόλοσ;	xn--nxasmq6b
+B;	Xn--Nxasmq6b;	βόλοσ;	xn--nxasmq6b
+T;	Βόλος;	βόλος;	xn--nxasmq6b
+N;	Βόλος;	βόλος;	xn--nxasmm1c
+T;	www.ශ\u0DCA\u200Dර\u0DD3.com;	;	www.xn--10cl1a0b.com
```
Comment (joshfree) [inline src/System.Globalization.Extensions/tests/IdnMapping/Data/Unicode_6_0/IdnaTest.txt]: It appears the formatting has changed in this file.  See Line 188+ as examples where tabs appear (at least via github) to have been removed

**Comment by joshfree** [comment]
please do a sanity check between the git commit and the TFS side to ensure no file encoding issues have crept into the data/test files.

**Comment by Priya91** [comment]
Checked this with the internal TFS version. The diff is the same.

**Comment by Priya91** [comment]
The diff matches in my local files from TFS and github match.

**Pull request merged**